features/configuration/output_stream.feature

```diff
@@ -23,5 +23,5 @@ Feature: Custom output stream
         end
       end
       """
-    When I run `rspec spec/example_spec.rb`
+    When I run `rspec spec/example_spec.rb --format progress`
```

Comment:
This is unrelated as far as I can tell and should be reverted, if it's not unrelated please explain why it's here and it'd be better to have a whole scenario for it.